components/telemetry-operator/internal/fluentbit/config_builder_test.go

```diff
@@ -337,3 +341,44 @@ func TestResolveSecretValue(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, resolved, "${PIPELINE_TEST_NAMESPACE_TEST_NAME_TEST_KEY}")
 }
+
+func TestLokiOutputPlugin(t *testing.T) {
```

Comment:
Let's also add a test for the `MergeSectionsConfig`, similar to the custom and http output